packages/viewer/src/properties.ts

```diff
@@ -71,31 +81,34 @@ export class Properties {
       const urlParams = new URLSearchParams(params);
 
       if (urlParams.get('dpi') !== null && urlParams.get('dpi') !== undefined) {
```

Comment:
Se que no es cosa de este PR pero la función `get` del objeto `URLSearchParams`  no devuelve `undefined`, sino `null` en el caso de que no exista la clave que se busca ([documentación](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/get)). Los ifs por lo tanto podrían ahorrarse la comprobación se si es undefined. Además, los ifs se podrían limitar simplemente a 


```
if(urlParams.get('element')) { ... }
``` 

o mejor aun, usar la función [has](https://developer.mozilla.org/en-US/docs/Web/API/URLSearchParams/has)

´´´
if(urlParams.has('element')) { ... }
´´´

Reply:
Will work on this next week